src/components/Timer/timer.test.ts

```diff
@@ -0,0 +1,37 @@
+// import { Challenge } from '../../model/ChallengeConfiguration'
+/**
+ * import gpx path data
+ * read data from file and get string from data
+ * convert string data to be a TrackPath
+ */
+const testTimer = test('returns time from component', () => {
```

Comment:
In Jest, we don't want to declare a function like this. In fact, we don't need to! Jest will automatically search for test files and run them during test-time.

Instead of:
➖️ `const testTimer = test(...`
We should delete the function and use:
➕️ `test(...`

Since we're deleting the function declaration here, we also shouldn't export anything at the bottom.